optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -130,6 +138,22 @@
             `(batch_size, num_heads, encoder_sequence_length, embed_size_per_head)`.
 """
 
+VISION_ENCODER_DECODER_SEQ2SEQ_ONNX_MODEL_DOCSTRING = r"""
+    Args:
+        pixel_values (`torch.FloatTensor`):
+            Featuers extracted from an Image.
+            `(batch_size, num_channels, height, width)`.
+        decoder_input_ids (`torch.LongTensor`):
+            Indices of decoder input sequence tokens in the vocabulary of shape `(batch_size, decoder_sequence_length)`.
+        encoder_outputs (`torch.FloatTensor`):
+            The encoder `last_hidden_state` of shape `(batch_size, encoder_sequence_length, hidden_size)`.
+        past_key_values (`tuple(tuple(torch.FloatTensor), *optional*)`
```

Comment:
```suggestion
        past_key_values (`tuple(tuple(torch.FloatTensor), *optional*)`
```

We should not put `*optional*` in Optimum following this discussion: https://huggingface.slack.com/archives/C02P0559X9S/p1669628754896019

Can rather precise what the default is? Although, for the type: `Tuple[Tuple[torch.FloatTensor]]`

(edit: this is copy paste, so could you edit it as well in the rest?)

Reply:
Added default to None as in the function sig